machine_learning/linear_discriminant_analysis.py

```diff
@@ -293,7 +331,9 @@ def main():
         print("-" * 100)
 
         # Calculating the value of probabilities for each class
-        probabilities = []  # An empty list to store values of probabilities for each class
+        probabilities = (
+            []
+        )  # An empty list to store values of probabilities for each class
```

Comment:
Put this comment on a separate line above so that black does not make your code ugly when it reformats.